packages/engine/Source/Scene/LabelCollection.js

```diff
@@ -183,6 +189,7 @@ function rebindAllGlyphs(labelCollection, label) {
       character,
       label._fontFamily,
       label._fontStyle,
+      label._fontSize,
```

Comment:
TODO: This line was included to force updates in the texture atlas, but managing the glyph cache correctly would take more investigation.